app/bundles/SmsBundle/Sms/TransportChain.php

```diff
@@ -80,6 +80,11 @@ private function getPrimaryTransport()
     {
         $enabled = $this->getEnabledTransports();
 
+        // If there no primary transport selected and there is just one available we will use it as primary
+        if (count($enabled) == 1) {
```

Comment:
Please use typesafe comparisons

Reply:
how does it influence anything?

Reply:
It shouldn't, but it is still best practice to always use typesafe comparisons wherever possible as they are (minimally) faster and simply for clean code purposes.